### PR TITLE
Table 1a - added chart

### PR DESCRIPTION
**Pull request by minotogna**

### CODE REVIEW

File: common/objectUtils.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const R = require('ramda')
+
+const types = {
+  fn: 'Function',
+}
+
+const _isType = type => R.pipe(R.type, R.equals(type))
+
+const isFunction = _isType(types.fn)
```
Comment (sorja) [inline common/objectUtils.js]: typeof() ?

File: webapp/app/assessment/components/dataTable/table/row.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ const Row = props => {
     return null
   }
 
-  const renderer = type === 'custom' ? render : rowRenderers[type]``
+  const renderer = type === 'custom' ? render : rowRenderers[type]
```
Comment (sorja) [inline webapp/app/assessment/components/dataTable/table/row.js]: maybe types.custom?

File: common/objectUtils.js (hunk 1 above)
Comment (minotogna) [reply common/objectUtils.js]: Ramda has more specific object types (like array, null). I would prefer Ramda

Comment (sorja) [reply common/objectUtils.js]: ok

Comment (minotogna) [reply common/objectUtils.js]: https://ramdajs.com/docs/#type

File: webapp/app/assessment/components/dataTable/table/row.js (hunk 2 above)
Comment (minotogna) [reply webapp/app/assessment/components/dataTable/table/row.js]: removed completely. that was useless. we don't support custom type

**Pull request merged**